specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/MigrateFromSmartDetections.json

```diff
@@ -0,0 +1,273 @@
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection": {
+      "post": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Starts an asynchronous migration process of Smart Detection to smart alerts in an Application Insights resource.",
+        "operationId": "MigrateFromSmartDetection_StartMigration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SmartDetectionMigrationRequest"
+            },
+            "description": "The parameters controlling the migration process."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Migration request was accepted. Processing will start promptly.",
+            "schema": {
+              "$ref": "#/definitions/MigrationStatusResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MigrationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Migration Triggering": {
+            "$ref": "./examples/migrateFromSmartDetectionStartMigration.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection/{migrationId}": {
+      "get": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Retrieve the status of a migration process.",
+        "operationId": "MigrateFromSmartDetection_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/MigrationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MigrationStatusResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MigrationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Migration Status": {
+            "$ref": "./examples/migrateFromSmartDetectionGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SmartDetectionMigrationRequest": {
+      "required": [
+        "scope"
+      ],
+      "properties": {
```

Comment:
>"properties" [](start = 6, length = 12)

Are all your properties optional? otherwise add "required" like you have done below.

Reply:
yes, only scope is mandatory in the request